frontend/packages/kubevirt-plugin/src/constants/v2v-import/ovirt/ovirt-network-interface-model.ts

```diff
@@ -3,18 +3,15 @@ import { ObjectEnum } from '../../object-enum';
 import { NetworkInterfaceModel } from '../../vm/network';
 
 export class OvirtNetworkInterfaceModel extends ObjectEnum<string> {
-  static readonly E1000 = new OvirtNetworkInterfaceModel('e1000', NetworkInterfaceModel.E1000);
+  static readonly E1000 = new OvirtNetworkInterfaceModel('e1000', NetworkInterfaceModel.E1000E);
   static readonly PCI_PASSTHROUGH = new OvirtNetworkInterfaceModel(
```

Comment:
let's wait first until discussion in https://bugzilla.redhat.com/show_bug.cgi?id=1869208 is resolved

Reply:
ok

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1869208 is resolved as wont fix.
a. the API will continue to accept the unsupported cards,
b. Once Import operator will merge (or not) a fix that will map unsupported network cards, we will could fix our mapping accordingly.